### PR TITLE
Add orange mobile money logo

### PR DESCRIPTION
**Pull request by tolu-paystack**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by hellicarusprime** [comment]
@tolu-paystack Can you please update the name in the YML file? It shouldn't have any special characters, including dashes, spaces or underscores.

**Comment by tolu-paystack** [comment]
@hellicarusprime changes made, please re-review

**Review by hellicarusprime — APPROVED** [review]

**Comment by adeniyiao** [comment]
Hi @tolu-paystack ,  could you please rebase your branch `tolu-paystack:orange_mobilemoney` with `activemerchant/master ` to resolve the conflicts. 

Thanks

**Comment by tolu-paystack** [comment]
@adeniyiao done

File: db/payment_icons.yml [hunk 1]
```diff
@@ -2190,3 +2190,7 @@
   name: bancomat
   label: Bancomat
   group: other
+-
+name: orangemobilemoney
```
Comment (adeniyiao) [inline db/payment_icons.yml:2194]: Hi @tolu-paystack , the name here is not correctly formated. Kindly address it. 

Thanks

Comment (tolu-paystack) [reply db/payment_icons.yml:2194]: Hey @adeniyiao, what's incorrect about the formatting? [An earlier comment](https://github.com/activemerchant/payment_icons/pull/1131#issuecomment-1932032566) said the name can't have dashes, spaces, or underscores, so I'm not sure how else to write it

Comment (adeniyiao) [reply db/payment_icons.yml:2194]: > Hey @adeniyiao, what's incorrect about the formatting? [An earlier comment](https://github.com/activemerchant/payment_icons/pull/1131#issuecomment-1932032566) said the name can't have dashes, spaces, or underscores, so I'm not sure how else to write it

Apologies if I wasn't clear. It's the position of the `name` which is not aligned in line 2194

Screenshot 2024-04-03 at 10 00 59 AM

Comment (tolu-paystack) [reply db/payment_icons.yml]: Got it, thanks @adeniyiao. This is fixed now

**Pull request merged**